source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -110,9 +110,38 @@ To enable root login via SSH, perform the following steps:
    .. code-block:: bash
 
       mkdir /etc/ssh
+      
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
+   
+#. Set the configuration variable in /etc/ssh/sshd_config
+
+   .. code-block:: console
+    
+      PermitRootLogin yes
+      
+Enable X11-forwarding 
+*****************
+
+X11 forwarding allows you to securely run graphical applications (i.e., X clients)
+over the ssh conection. this will alow for remote gui apps without the need for full
+VNC/remotedesktop.
+To enable X11-forwarding via SSH, perform the following steps:
 
-#. Set the configuration variable.
+#. Create a *ssh* directory in :file:`/etc`, if it does not already exist.
 
    .. code-block:: bash
 
-      echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
+      mkdir /etc/ssh
+
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
```

Comment:
@odwdinc, @puneetse.  Thank you for your work on this.  Please hold on any further edits.  https://github.com/clearlinux/clear-linux-documentation/pull/277 removes this link, and this document, from the bundles page.  We need to discuss whether this file merits being a standalone document in our entire doc set. Please reply here if you are available for a quick meeting Mon. or Tues. to discuss.  Otherwise, I raise it in Wed. meeting.

Reply:
Yes, anytime works for me. Thanks.

Reply:
I could do 10am to 1pm tomorrow Tuesday